src/clj/puppetlabs/services/jruby/jruby_puppet_service.clj

```diff
@@ -5,6 +5,7 @@
             [puppetlabs.services.jruby-pool-manager.jruby-core :as jruby-core]
             [puppetlabs.trapperkeeper.core :as trapperkeeper]
             [puppetlabs.trapperkeeper.services :as tk-services]
+            [puppetlabs.i18n.core :as i18n :refer [trs tru]]
```

Comment:
Shoot, another upstream merge conflict resolution issue. It was used initially. I'll remove it right now.